tests/TestCase/ExceptionsTest.php

```diff
@@ -69,13 +69,6 @@ public function testFatalErrorException()
         $this->assertSame(__FILE__, $exception->getFile());
         $this->assertSame(1, $exception->getLine());
         $this->assertSame($previous, $exception->getPrevious());
-
-        $exception = new FatalErrorException('message', null, __FILE__, 1, $previous);
```

Comment:
This is the same test as above but checking the default error code. There is no need to do this for a random exception.